data/presets/X-Pressive/X-Pressive1.xpf

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!DOCTYPE lmms-project>
+<lmms-project version="1.0" creator="LMMS" creatorversion="1.2.0-rc2.6" type="instrumenttracksettings">
+  <head/>
+  <instrumenttracksettings muted="0" type="0" name="X-Pressive" solo="0">
```

Comment:
Tell you the truth, I'm not sure that those presets should be merge. And even if they do, I think they should have more meaningful naming... I'm not so good at naming sounds so... if someone volunteers... 
Also if someone made nice presets which he wants them to be merged...